draft-packages/stories/FilterDrawer.stories.scss

```diff
@@ -9,3 +11,9 @@
     width: auto;
   }
 }
+
+.siteDemoWrapper {
+  background-color: $kz-color-wisteria-700;
+  height: 368px;
```

Comment:
Without the height specified, we could reuse this class for other components that want a Wisteria background. Perhaps just leave a comment that says this is for Filter Drawer so if we want to abstract it out later, it's easy?